### PR TITLE
Add tooltip

### LINKED ISSUE #244: Meet your reps copy/style changes
From [docs](https://docs.google.com/document/d/18H9k5umSZwnbK_qDiaNSOhrvdT1c7r77F56sL78_a3w/edit)
>Insert a sentence under the cards explaining what an LCV score is, in small text: 
>The “LCV” number is from the League of Conservation Voters’ National Environmental Scorecard, which evaluates whether lawmakers take favorable votes on environmental issues. The closer to 100, the better. Shown above are your reps’ [LIFETIME / 2020 – whichever one it is ] scores.
>Use tooltip

### PR DESCRIPTION
**Pull request by jacknugent**
Closes #244

### CODE REVIEW

**Comment by jacknugent** [comment]
Thoughts on tooltip? Little verbose:

image

**Comment by kevincianfarini** [comment]
I agree. I think it's a bit too much text. I will reach out to Eliza.

File: frontend/src/common/Components/LegislatorCard/LegislatorCard.tsx [hunk 1]
```diff
@@ -42,14 +44,28 @@ export default function LegislatorCard({ legislator, call }: Props) {
...
-                            <div
-                                className={`${styles.lcvScoreContainer} d-flex justify-content-center align-items-center flex-row`}
+                            <OverlayTrigger
+                                placement="bottom"
+                                overlay={
+                                    <Tooltip className="position-absolute" id={`lcv-${legislator.bioguideId}`}>
+                                        The “LCV” number is from the League of Conservation Voters’ National
+                                        Environmental Scorecard, which evaluates whether lawmakers take favorable votes
+                                        on environmental issues. The closer to 100, the better. Shown above are your
+                                        reps’ <b>{legislator.lcvScores[0].scoreType.type}</b> scores.
```
Comment (kevincianfarini) [inline frontend/src/common/Components/LegislatorCard/LegislatorCard.tsx:58]: I don't think this necessarily captures all of the information we're trying to convey. We want it to either list `lifetime` or the year in question if it's a yearly score. For example, `2020`.

**Comment by kevincianfarini** [comment]
Final verbiage: 

> The League of Conservation Voters (LCV) calculates a score of 0-100 for favorable environmental votes in Congress. Shown above is the {lifetime | 2020} score.

**Comment by jacknugent** [comment]
Done

**Review by kevincianfarini — APPROVED** [review]

**Pull request merged**